proto/cosmos/distribution/v1beta1/tx.proto

```diff
@@ -29,7 +29,12 @@ service Msg {
 
   // FundCommunityPool defines a method to allow an account to directly
   // fund the community pool.
-  rpc FundCommunityPool(MsgFundCommunityPool) returns (MsgFundCommunityPoolResponse);
+  //
+  // DEPRECATED: Use x/pool modules FundCommunityPool instead.
```

Comment:
Please use `Deprecated:` not `DEPRECATED:` as the former is what go tooling understands.